### PR TITLE
Python patches for Feb 2022 update

### PR DESCRIPTION
**Pull request by johnwason**
This pull request contains patches for the Feb 2022 python update.

### CODE REVIEW

File: tesseract_geometry/include/tesseract_geometry/geometry.h [hunk 1]
```diff
@@ -50,6 +50,7 @@ enum GeometryType
   CONVEX_MESH,
   SDF_MESH,
   OCTREE,
+  POLYGON_MESH
 };
 static const std::vector<std::string> GeometryTypeStrings = {
   "SPHERE", "CYLINDER", "CAPSULE", "CONE", "BOX", "PLANE", "MESH", "CONVEX_MESH", "SDF_MESH", "OCTREE"
```
Comment (Levi-Armstrong) [inline tesseract_geometry/include/tesseract_geometry/geometry.h:56]: I would also append "POLYGON_MESH"

File: tesseract_geometry/include/tesseract_geometry/impl/polygon_mesh.h [hunk 2]
```diff
@@ -201,7 +203,7 @@ class PolygonMesh : public Geometry
    *
    * @return The vertex normal vector
    */
-  std::shared_ptr<const tesseract_common::VectorVector3d> getNormals() const { return normals_; }
+  const std::shared_ptr<const tesseract_common::VectorVector3d>& getNormals() const { return normals_; }
```
Comment (Levi-Armstrong) [inline tesseract_geometry/include/tesseract_geometry/impl/polygon_mesh.h:206]: Just curious, why the switch to const&?

File: tesseract_geometry/include/tesseract_geometry/geometry.h (hunk 1 above)
Comment (Levi-Armstrong) [reply tesseract_geometry/include/tesseract_geometry/geometry.h]: I made the change.

File: tesseract_geometry/include/tesseract_geometry/impl/polygon_mesh.h (hunk 2 above)
Comment (johnwason) [reply tesseract_geometry/include/tesseract_geometry/impl/polygon_mesh.h:206]: SWIG wasn't detecting the type properly without const&

**Comment by johnwason** [comment]
I am still cleaning up these patches to fix the CI errors

**Comment by Levi-Armstrong** [comment]
@johnwason You good with me merging this?

**Comment by johnwason** [comment]
Yeah looks ready

**Pull request merged**